### ISSUE TITLE
image case sensitive issue

### ISSUE THREAD

**Issue by KAnderson** [mfractor/mfractor-feedback#294]
1. Create a XF app
2. Import an image using image tools (or manually)
3. Reference Image but use upper case letter where there should be a lower case

![image](https://user-images.githubusercontent.com/273611/90444794-c5204300-e09b-11ea-89e6-4f1611c6cf4f.png)

In this screenshot you can see that the image is not found, but if I changed "Calendar.png" to "calendar.png" it does work. I also did not receive the name suggestion as expected.

https://github.com/KAnderson/CollectionViewExample.git

##Installation Information

```
--Platform Information--
Operating System: Windows
OS: Win32NT
OS Service Pack: 
OS Version: Microsoft Windows NT 10.0.19042.0
--MFractor Information--
Product Name: MFractor
Product Version: 4.3.0.0
Product Variant: Visual Studio For Windows
Product Variant Version: 16.0.0.0
Product SKU: MFRACTOR-VS-WINDOWS
Commit SHA: 456a155ffb822243ae788e1ff912fa28be4af2e3
Revision: 4600
Branch: master
Date: 06/08/2020 2:26:45 AM (UTC)
Agent: Local
Installed extensions:
	
```

**Comment by matthewrdev** [comment]
@KAnderson I can reproduce this issue and am investigating.

**Comment by matthewrdev** [comment]
Resolved in MFractor 4.3.1.
